streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -137,62 +142,57 @@ private InternalMockProcessorContext getProcessorContext(final RecordingLevel re
         return getProcessorContext(streamsProps);
     }
 
-    @After
-    public void tearDown() {
-        rocksDBStore.close();
-    }
-
     @Test
     public void shouldAddStatisticsToInjectedMetricsRecorderWhenRecordingLevelIsDebug() {
         final RocksDBStore store = getRocksDBStoreWithRocksDBMetricsRecorder();
-        final InternalMockProcessorContext mockContext = getProcessorContext(RecordingLevel.DEBUG);
+        context = getProcessorContext(RecordingLevel.DEBUG);
```

Comment:
I left it out intentionally because we always cause `#close` on this `rocksDBStore` and this causes this test to fail.

It's just because of the mock, so we could instead add some `expect...` lines at the end of the test, but it seemed cleaner to just use a local `RocksDBStore` here